config/media_store.sql

```diff
@@ -212,7 +210,7 @@ INSERT INTO Artist (ArtistId, Name) VALUES (18, 'Chico Science & Nação Zumbi')
 INSERT INTO Artist (ArtistId, Name) VALUES (19, 'Cidade Negra');
 INSERT INTO Artist (ArtistId, Name) VALUES (20, 'Cláudio Zoli');
 INSERT INTO Artist (ArtistId, Name) VALUES (21, 'Various Artists');
-INSERT INTO Artist (ArtistId, Name) VALUES (22, 'Led Zeppeli');
+INSERT INTO Artist (ArtistId, Name) VALUES (22, 'Led Zeppelin');
```

Comment:
I can't believe these errors have been in the script for four years! Well spotted.